### PR TITLE
Us24 controller manager testimony validation

### PR DESCRIPTION
**Pull request by Vannou28**

### CODE REVIEW

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

File: src/Model/TestimoniesManager.php [hunk 1]
```diff
@@ -38,4 +38,14 @@ public function update(array $item): bool
 
         return $statement->execute();
     }
+    /**
+     * Update testimony in database
+     */
+    public function updateValidate(array $item): bool
```
Comment (LucyMerlier) [inline src/Model/TestimoniesManager.php:44]: updateValidate? that method name is not really self explanatory + your parameter is called "item"? not self explanatory either

File: src/Controller/AdminController.php [hunk 2]
```diff
@@ -51,4 +51,13 @@ public function addService(): string
...
         return $this->twig->render('Admin/Services/add_service.html.twig');
     }
+    public function testimonyaddValidation(): void
+    {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $testimony = array_map('trim', $_POST);
+            $testimoniesManager = new TestimoniesManager();
+            $testimoniesManager->updateValidate($testimony);
+            header('Location: /Admin/index');
+        }
```
Comment (LucyMerlier) [inline src/Controller/AdminController.php:61]: not form validation?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Model/TestimoniesManager.php (hunk 1 above)
Comment (Vannou28) [reply src/Model/TestimoniesManager.php]: a news for the modifications of the items

File: src/Controller/AdminController.php (hunk 2 above)
Comment (Vannou28) [reply src/Controller/AdminController.php]: new PR for that

**Comment by RaphaelBS-WCS** [comment]
changes not needed.

**Pull request closed without merging**